x/circuit/autocli.go

```diff
@@ -37,16 +37,14 @@ func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
 			RpcCommandOptions: []*autocliv1.RpcCommandOptions{
 				{
 					RpcMethod: "AuthorizeCircuitBreaker",
-					Use:       "authorize [grantee] [permissions_json] --from [granter]",
+					Use:       "authorize [grantee] [level] [msg_type_urls] --from [granter]",
```

Comment:
Is this diff correct?

Reply:
yes, before you had to submit a json (awful UX), now autocli can use inner field, allowing you directly to set those values. Which is immediately usable in circuit